src/pages/contact.tsx

```diff
@@ -1,8 +1,15 @@
 import { PageProps } from 'gatsby'
 import React from 'react'
+import SEO from '../components/SEO'
+import Layout from '../layout/Layout'
 
 const ContactPage: React.FC<PageProps> = () => {
-  return <div>asd</div>
+  return (
+    <Layout>
+      <SEO />
+      <div>asd</div>
```

Comment:
Is `asd` intentional? I think a "TODO" would be less cringe if we leave it in by mistake and some IDE can highlight left in `//TODO` comments

Reply:
Applied it in new commit.

Also, some browsers do not emit perfectly the scrollTop property of the window element (optimization I guess), so yeah, one can f*ck up the navbar by dragging it crazy.